### PR TITLE
User component switch

### PR DESCRIPTION
**Pull request by soeren-schmaljohann-2denker**

### CODE REVIEW

File: packages/apptive_grid_core/lib/src/network/authentication/web_auth_enabler/configure_web.dart [hunk 1]
```diff
@@ -12,7 +12,7 @@ Future<void> enableWebAuth(ApptiveGridOptions options) async {
   final url = window.location.href;
   if (url.contains('state=')) {
     final issuerUri = Uri.parse(
-      'https://iam.zweidenker.de/auth/realms/${options.environment.authRealm}',
+      '${options.environment.url}/auth/${options.authenticationOptions.authGroup}/authorize',
     );
     final issuer = await openid.Issuer.discover(issuerUri);
```
Comment (ABausG) [inline packages/apptive_grid_core/lib/src/network/authentication/web_auth_enabler/configure_web.dart:17]: This issuer should be the same as in the authenticator. So maybe it's worthe to extract it into a variable

File: packages/apptive_grid_core/test/authenticator_test.dart [hunk 2]
```diff
@@ -451,8 +417,7 @@ void main() {
       await authenticator.authenticate();
       final launchedUrl = await completer.future;
       expect(
-        launchedUrl
-            .startsWith('https://iam.zweidenker.de/auth/realms/apptiveGrid/'),
+        launchedUrl.startsWith('https://app.apptivegrid.de/auth/apptivegrid'),
```
Comment (ABausG) [inline packages/apptive_grid_core/test/authenticator_test.dart:420]: should include `/authorize` in the path

Comment (soeren-schmaljohann-2denker) [reply packages/apptive_grid_core/test/authenticator_test.dart]: done, but it should be `/token`

File: packages/apptive_grid_core/lib/src/network/authentication/web_auth_enabler/configure_web.dart (hunk 1 above)
Comment (soeren-schmaljohann-2denker) [reply packages/apptive_grid_core/lib/src/network/authentication/web_auth_enabler/configure_web.dart]: done

**Review by ABausG — APPROVED** [review]

**Pull request merged**